velox/docs/monitoring/stats.rst

```diff
@@ -65,3 +65,7 @@ These stats are reported only by HashBuild and HashAggregation operators.
    * - hashtable.numTombstones
      -
      - Number of tombstone slots in the hash table.
+   * - hashtable.buildWallNanos
```

Comment:
This stat is reported only by the HashBuild operator, right? HashAggregation operator never reports it. Let's make this clear.

Reply:
Fixed